challenges/left_join/left_join.py

```diff
@@ -0,0 +1,22 @@
+from hash_table import HashTable
+
+
+def left_join(hash1, hash2):
+    """
+    This functions takes two hashmaps with word strings as
+    keys and performs a left join into a list.
+    """
+
+    joined = HashTable()
+
+    keys1 = hash1.__iter__()
```

Comment:
You should not directly call magic methods like this. `__iter__()` allows you to iterate over a collection using a `for` loop.